code/clement/html_css_flask/lab03/static/styles.css

```diff
@@ -0,0 +1,66 @@
+body {
+    background-image: url(https://c.tenor.com/AJBv6U1rfkQAAAAC/background-image-galaxy.gif);
+    margin: 0px;
```

Comment:
A lot of this CSS can be replaced with [bootstrap](https://getbootstrap.com/docs/5.1/utilities/spacing/). For example `marin: 0px;` could be `m-0` as a class.